template_init_data.json

```diff
@@ -0,0 +1,34 @@
+{
+    "items": [
+        {
+            "key": "/events/DY1cscbWCk3F4_ZUakA3B",
+            "data": {
+                "id": "DY1cscbWCk3F4_ZUakA3B",
+                "title": "Yoga Class",
+                "start": "2019-12-20",
```

Comment:
AFAICT it's not possible to have events without time, or events without date, so this initial data is somewhat confusing.